apps/workflows/serializers.py

```diff
@@ -145,6 +145,7 @@ class RejectSerializer(serializers.Serializer):
 class WorkflowModelSerializer(serializers.ModelSerializer):
 
     classification = serializers.SerializerMethodField()
+    task_key = serializers.CharField(read_only=True)
```

Comment:
This should match the null/blank settings with database model, see https://github.com/RedHatProductSecurity/osidb/pull/929/commits